source/platform.js

```diff
@@ -208,22 +210,23 @@ EcobeePlatform.prototype.update = function (callback) {
 
 EcobeePlatform.prototype.sensors = function (reply) {
   this.log.debug("Setting values of sensors...");
-  if (!reply.thermostatList || reply.thermostatList.length == 0) {
+  if (!reply.thermostatList || reply.thermostatList.length === 0) {
     this.log.error("No Ecobee 3 thermostats found. Please, make soure your thermostat is registered.");
     return;
   }
 
   for (var thermostatConfig of reply.thermostatList) {
     for (var sensorConfig of thermostatConfig.remoteSensors) {
-      sensorConfig.code = sensorConfig.code || thermostatConfig.identifier; // Hack around missing code for the thermostat itself
+      if (sensorConfig.type !== 'ecobee3_remote_sensor') continue
```

Comment:
i think that's fine, though i'd flip it to be something like:

        "exclude_thermostat": false

as the default. so we get the old behavior by default.